.github/workflows/update-vscode-dvc.yml

```diff
@@ -0,0 +1,31 @@
+name: Submodule Updates
+
+on:
+  push:
+    branches-ignore: [main]
```

Comment:
Wouldn't this be triggered on push on any other branches?

Reply:
yeah, this doesn't work yet